src/v2/view-builder/internals/BaseForm.js

```diff
@@ -62,14 +62,12 @@ export default Form.extend({
     }
   },
 
-  addCallouts () {
-    const warningMsgs = this.options.appState.get('messages');
-    if (warningMsgs && warningMsgs.value.length) {
-      const messageCallout = createCallout({
-        content: warningMsgs.value[0].message,
-        type: 'warning',
-      });
-      this.add(messageCallout, '.o-form-error-container');
+  showMessages () {
+    // render messages as text
+    const messagesObj = this.options.appState.get('messages');
+    if (messagesObj && messagesObj.value.length && messagesObj.value[0].class === 'INFO') {
+      const content = messagesObj.value[0].message;
+      this.add(`<span class="terminal-content">${content}</span>`, '.o-form-content');
```

Comment:
probably change this to `.o-form-terminal-message-container`

Reply:
by the way this can still live inside `o-form-error-container` right ?

Reply:
the class name probably shall not have `terminal` keyword which implies it's meant for `TerminalView` but this is just a generic handler for displaying messages.
maybe `o-form-ion-messages-container` ?

Reply:
We can also remove `o-form` from the className should we just call it `ion- messages-container`? 

I moved it out of `o-form-error-container` because messages are not just errors and it need not be within the `o-form-error-container`.

Reply:
I'm guessing this container will be add before form header because of using select `.o-form-content` ?

Reply:
<img width="1293" alt="Screen Shot 2020-07-01 at 9 53 08 PM" src="https://user-images.githubusercontent.com/23267876/86317832-481d3500-bbe5-11ea-9281-28aed8a465ae.png">

Reply:
that would work well only for terminal view or view that has no inputs (ui schema)
kind of defeat the purpose of being generic. 😅

Reply:
btw @vincentvoong-okta confirmed that we want to remove the heading title from terminal view, so I will remove that in my next commit.

Reply:
I put it back under `o-form-error-container` for now.